plugin/fzf.vim

```diff
@@ -334,19 +334,20 @@ function! fzf#wrap(...)
 endfunction
 
 function! s:use_sh()
-  let [shell, shellslash] = [&shell, &shellslash]
+  let [shell, shellslash, shellcmdflag] = [&shell, &shellslash, &shellcmdflag]
   if s:is_win
     set shell=cmd.exe
+    set shellcmdflag=/c
```

Comment:
Setting `shell` options have too many edge cases on Windows because there are 3 incompatible shells to work with and fzf's Vim plugin supports neovim. Like vim-plug, don't set any shell options on Windows.

Reply:
Not setting shell options here will take a lot of work so consider Neovim's default shell options here.

```vim
let &shellcmdflag = has('nvim') ? '/s /c' : '/c'
```

Reply:
Thank you for your advice.
I applied it at abde45180f947f5b6f1d3320973d9a7025fc8575.